### ISSUE TITLE
In storefront, implement top-right links as cms

### ISSUE THREAD

**Issue by Xymmer** [SAP/cloud-commerce-spartacus-storefront#353]
Depends on #352.

To do:

- Replace Sale/Contactus/Help with the cms equivalents.
- Remove hardcoded selectors currently there.
- Create a page for each link (sale, contact us, help).

Acceptance criteria:
- Can see the 3 links in same place in Spartacus.'
- Clicking the links opens the correct page and displays the dummy text we pulled from cms.

**Comment by tobi-or-not-tobi** [comment]
The pages related to those links are not yet solved, @Xymmer can you create a separate ticket for this?

**Comment by Xymmer** [comment]
new issue is #935

**Comment by tobi-or-not-tobi** [comment]
No QA, can only be tested when #935 is done.
